packages/frontend/src/retro/hooks/useTimedEffect.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+
+interface useFinishEffectProps {
+  effectLength: number;
+}
+export default function useTimedEffect({ effectLength }: useFinishEffectProps) {
+  const [isEffectActive, setIsEffectActive] = useState(false);
+
+  useEffect(() => {
+    if (isEffectActive) {
+      const timeout = setTimeout(() => {
+        setIsEffectActive(false);
+      }, effectLength);
+      return () => {
+        clearTimeout(timeout);
+      };
+    }
+  }, [isEffectActive, effectLength]);
+
+  function startEffect() {
+    if (!isEffectActive) {
+      setIsEffectActive(true);
+    }
```

Comment:
This check is not neccessary, since we dont rerender if isEffectActice didnt change. So setting it from true to true does nothing.